content/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,528 @@
+    Operation:    Update
+    Time:         2024-03-06T17:58:51Z
+    API Version:  cert-manager.io/v1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        f:revision:
+    Manager:      cert-manager-certificates-issuing
+    Operation:    Update
+    Subresource:  status
+    Time:         2024-03-06T17:58:52Z
+    API Version:  cert-manager.io/v1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:status:
+        .:
+        f:conditions:
+          .:
+          k:{"type":"Ready"}:
+            .:
+            f:lastTransitionTime:
+            f:message:
+            f:observedGeneration:
+            f:reason:
+            f:status:
+            f:type:
+        f:notAfter:
+        f:notBefore:
+        f:renewalTime:
+    Manager:         cert-manager-certificates-readiness
+    Operation:       Update
+    Subresource:     status
+    Time:            2024-03-06T17:58:52Z
+  Resource Version:  2128063
+  UID:               977eaa8a-1612-489b-a34d-0e78ab113096
+Spec:
+  Common Name:  example.com
+  Dns Names:
+    green.nginx.example.com
+    blue.nginx.example.com
+    nginx.example.com
+    example.com
+  Is CA:  true
+  Issuer Ref:
+    Kind:       Issuer
+    Name:       cfe-selfsigned-issuer
+  Secret Name:  cfe-tls-key-pair
+Status:
+  Conditions:
+    Last Transition Time:  2024-03-06T17:58:52Z
+    Message:               Certificate is up to date and has not expired
+    Observed Generation:   1
+    Reason:                Ready
+    Status:                True
+    Type:                  Ready
+  Not After:               2024-06-04T17:58:52Z
+  Not Before:              2024-03-06T17:58:52Z
+  Renewal Time:            2024-05-05T17:58:52Z
+  Revision:                1
+Events:                    <none>
+```
+
+### Deploy sample Nginx applications
+
+In order to configure and validate the Ingress TLS termination, three sample Nginx applications will be deployed to the cluster using the YAML manifest files from the GitHub repo [ingress-demo](https://github.com/GuopingJia/ingress-demo):
+
+```shell
+$ tree ingress-demo/
+ingress-demo/
+├── apps
+│   ├── nginx-blue.yaml
+│   ├── nginx-green.yaml
+│   └── nginx-main.yaml
+├── ingress-host-based-selfsigned.yaml
+├── ingress-path-based-selfsigned.yaml
+└── README.md
+```
+
+Each YAML manifest file in the folder *'apps'* defines the *Deployment* and the *Service* resource.  
+
+T﻿ype the following commands to deploy those Nginx applications to the namespace *nginx-apps*:
+
+```shell
+$ cd ingress-demo/
+$ kubectl apply -f apps/nginx-main.yaml -n nginx-apps
+service/nginx-main created
+deployment.apps/nginx-main created
+$ kubectl apply -f apps/nginx-green.yaml -n nginx-apps
+service/nginx-green created
+deployment.apps/nginx-green created
+$ kubectl apply -f apps/nginx-blue.yaml -n nginx-apps
+service/nginx-blue created
+deployment.apps/nginx-blue created
+```
+
+Type the command shown below to check the details of application deployment:  
```

Comment:
I think you're missing the word "each" just prior to "application deployment":
"...check the details of each application deployment:"?